doc/reference/hardware.rst

```diff
@@ -27,6 +27,12 @@ Known Limitations
 *****************
 Platforms with multiple PCI segments
 
+ACRN assumes the following conditions are satisfied from the Platform BIOS
+a) All the PCI device BARs should be assigned resources, including VF BARs if a device supports.
```

Comment:
Add a blank line before the first item in a list.

Reply:
Done.